projects/packages/sync/src/modules/class-woocommerce.php

```diff
@@ -668,8 +641,77 @@ public function get_order_item_by_ids( $ids ) {
 		$placeholders = implode( ',', array_fill( 0, count( $ids ), '%d' ) );
 
 		$query = "SELECT * FROM {$this->order_item_table_name} WHERE order_item_id IN ( $placeholders )";
+		if ( ! empty( $order ) ) {
```

Comment:
Is there any reason to not fetch the items in DESC order in all cases? So that we don't need to add the extra `$order` param.
In any case, if we decide to keep it like this lets at least check and sanitize it before using it, eg `$order = in_array( $order, [ 'ASC', 'DESC' ] ) ? $order : 'DESC'`;

Reply:
Thanks for the comment, `get_order_item_by_ids` is mainly used in checksum logic. We might be able to just use DESC order in all cases, but wanted to play it safe. 
What do you think is better?

Reply:
I'd keep it simple assuming this doesn't affect any consumers of this method, which I guess it shouldn't. I'm good with w/e you prefer as long as we sanitize it if we keep it like this.

Reply:
Let's keep it safe, I added the sanitization in [17e43c2](https://github.com/Automattic/jetpack/pull/41433/commits/17e43c294e9eb20a42c82275a4f35aa206897d46)